src/Compilers/CSharp/Portable/Parser/Lexer_RawStringLiteral.cs

```diff
@@ -330,38 +329,17 @@ private void AddMultiLineRawStringLiteralLineContents(
             currentLineWhitespace.Clear();
             ConsumeWhitespace(currentLineWhitespace);
 
-            if (!StartsWith(currentLineWhitespace, indentationWhitespace))
+            var (errorCode, errorArguments) = CheckForIndentationError<StringBuilder, StringBuilderCharHelper>(
+                currentLineWhitespace, indentationWhitespace, isBlankLine: SyntaxFacts.IsNewLine(TextWindow.PeekChar()));
+
+            if (errorCode != 0)
             {
-                // We have a line where the indentation of that line isn't a prefix of indentation
-                // whitespace.
-                //
-                // If we're not on a blank line then this is bad.  That's a content line that doesn't start
-                // with the indentation whitespace.  If we are on a blank line then it's ok if the whitespace
-                // we do have is a prefix of the indentation whitespace.
-                var isBlankLine = SyntaxFacts.IsNewLine(TextWindow.PeekChar());
```

Comment:
all this logic is now shared with the parser instead of each having to reeimplement it.